cmake/compilers/Clang.cmake

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+if(EMSCRIPTEN)
+  set(TBB_EMSCRIPTEN 1)
+  set(TBB_COMMON_COMPILE_FLAGS ${TBB_COMMON_COMPILE_FLAGS} -fexceptions)
```

Comment:
Is `-fexceptions` flag needed for libraries build or tests build?

Reply:
Yes, we need it for both